packages/esm-patient-medications-app/src/order-basket/order-basket-search/order-basket-search-results.component.tsx

```diff
@@ -0,0 +1,192 @@
+            </p>
+          </div>
+        </Tile>
+      )}
+      <hr className={`${styles.divider} ${isTablet ? `${styles.tabletDivider}` : `${styles.desktopDivider}`}`} />
+    </>
+  );
+}
+
+interface DrugSearchResultItemProps {
+  drug: Drug;
+  onSearchResultClicked: (searchResult: OrderBasketItem, directlyAddToBasket: boolean) => void;
+}
+
+const DrugSearchResultItem: React.FC<DrugSearchResultItemProps> = ({ drug, onSearchResultClicked }) => {
+  const isTablet = useLayoutType() === 'tablet';
+  const {
+    templates,
+    isLoading: isLoadingTemplates,
+    error: fetchingDrugOrderTemplatesError,
+  } = useDrugTemplate(drug?.uuid);
+  const { t } = useTranslation();
+  const config = useConfig() as ConfigObject;
+  const orderItems: Array<OrderBasketItem> = useMemo(
+    () =>
+      templates?.length
+        ? templates.map((template) => getTemplateOrderBasketItem(drug, config?.daysDurationUnit, template))
+        : [getTemplateOrderBasketItem(drug, config?.daysDurationUnit)],
+    [templates, drug, config?.daysDurationUnit],
+  );
+
+  const handleSearchResultClicked = (searchResult: OrderBasketItem, directlyAddToBasket: boolean) => {
+    onSearchResultClicked(searchResult, directlyAddToBasket);
+  };
+
+  return (
+    <>
+      {orderItems.map((orderItem, indx) => (
+        <ClickableTile
+          key={templates?.length ? templates[indx]?.uuid : drug?.uuid}
+          role="listitem"
+          className={isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`}
+          onClick={() => handleSearchResultClicked(orderItem, false)}
+        >
+          <div className={styles.searchResultTile}>
+            <div className={styles.searchResultTileContent}>
+              <p>
+                <strong>{drug?.display}</strong> {drug?.strength && <>&mdash; {drug?.strength}</>}{' '}
+                {drug?.dosageForm?.display && <>&mdash; {drug?.dosageForm?.display}</>}
+              </p>
+              {fetchingDrugOrderTemplatesError ? (
+                <p>
+                  <span className={styles.errorLabel}>
+                    {t('errorFetchingDrugOrderTemplates', 'Error fetching drug order templates')}
+                  </span>
+                </p>
+              ) : (
+                <p>
+                  {orderItem?.frequency?.value && <span className={styles.label01}>{orderItem?.frequency?.value}</span>}
+                  {orderItem?.route?.value && <span className={styles.label01}>&mdash; {orderItem?.route?.value}</span>}
+                </p>
+              )}
+            </div>
+            <Button
+              className={styles.addToBasketButton}
+              kind="ghost"
+              hasIconOnly={true}
+              renderIcon={(props) => <ShoppingCart size={16} {...props} />}
+              iconDescription={t('directlyAddToBasket', 'Immediately add to basket')}
+              onClick={() => handleSearchResultClicked(orderItem, true)}
```

Comment:
```suggestion
              onClick={() => handleSearchResultClicked(orderItem, true)}
              tooltipPosition="left"
              tooltipAlignment="end"
```

To avoid the tooltip getting cut off:

<img width="624" alt="Screenshot 2022-12-22 at 23 46 20" src="https://user-images.githubusercontent.com/8509731/209223368-5c9fcde4-b6cf-4aa4-b7d5-5556534a84da.png">

Reply:
Thanks a lot!
Duly noted.